publish.ps1

```diff
@@ -68,10 +68,12 @@ function do-package()
     Copy-Item -Force "$distDir\readme.txt"            $tempDir -ErrorAction Stop
 
     # Create archive
-    7za a -tzip -mx9 $outZip "$tempDir\*"
+    7z a -tzip -mx9 $outZip "$tempDir\*"
```

Comment:
Yes, but that might be because they have defined an alias. So we just need to make sure that it will work on AppVeyor and a normal user system.